src/pages/address/components/TokenRow.tsx

```diff
@@ -0,0 +1,281 @@
+import React from "react";
+import LinkText from "@components/commons/LinkText";
+import NumericFormat from "@components/commons/NumericFormat";
+import { Token } from "mockdata/TokenData";
+import { truncateTextFromMiddle } from "shared/textHelper";
+import AddressTokenTableTitle from "./AddressTokenTableTitle";
+
+export default function TokenRow({ data }: { data: Token }) {
+  return (
+    <div>
+      {/* desktop */}
+      <div className="hidden lg:block">
+        <div className="grid grid-cols-9 py-5">
+          <div>
+            <LinkText
+              label={data.asset}
+              href={`/address/${data.asset}`}
+              customStyle="tracking-[0.01em]"
+            />
+          </div>
+          <div>
+            <div className="text-white-50 tracking-[0.01em]">{data.type}</div>
+          </div>
+          <div>
```

Comment:
i noticed we have quite a number of these `divs` in this file, do we really need them in UI?

Reply:
updated to remove unused divs